### PR TITLE
Use oss-lib for configuration and routing

### PR DESCRIPTION
**Pull request by akscram**

### CODE REVIEW

File: security/config.py [hunk 1]
```diff
@@ -13,17 +13,111 @@
...
+                        },
+                        "port": {"type": "integer"},
+                    },
+                    "required": ["host"],
+                    "additionalProperties": False,
+                },
+            },
+        },
+        "required": ["hosts"],
+        "additionalProperties": False,
```
Comment (redixin) [inline security/config.py]: User may want to add some extra arguments like described here: https://elasticsearch-py.readthedocs.io/en/master/#ssl-and-authentication

File: security/config.py [hunk 2]
```diff
@@ -13,17 +13,111 @@
...
+                        "username": {
+                            "type": "string",
+                            "minLength": 1,
+                        },
+                        "password": {
+                            "type": "string",
+                            "minLength": 1,
+                        },
+                    },
+                    "required": ["auth_url", "username", "password"],
```
Comment (redixin) [inline security/config.py]: It wont work without "tenant_name"

File: security/config.py (hunk 1 above)
Comment (akscram) [reply security/config.py]: Sure, added these fields too.

File: security/config.py (hunk 2 above)
Comment (akscram) [reply security/config.py]: Done.

**Review by boris-42 — APPROVED** [review]

**Pull request merged**